runtime/avn/src/lib.rs

```diff
@@ -121,14 +130,38 @@ where
     }
 }
 
+pub struct WeightToFee;
+impl WeightToFeePolynomial for WeightToFee {
+    type Balance = Balance;
+    fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
+        // We adjust the fee conversion so that a simple token transfer
+        // direct to chain costs base_fee USD.
+        let min_fee: u128 = pallet_avn_transaction_payment::Pallet::<Runtime>::get_min_avt_fee();
+
+        // The magic number (2.380951) is the result of :
+        // setting p = 50 * MILLI_BASE, the cost of a simple transfer was 119.04775 milli AVT
+        // (visual observation on polkadot.js). magic_number = 119.04775 / 50 = 2.380951
+        let factor = FixedU128::saturating_from_rational(1_000_000u128, 2_380_951u128);
```

Comment:
Can we check if the cost of a simple transfer is actually 2.380951 on this chain based on the formula above.